### DISCUSSION TITLE
Ignore markup when comparing HTML

### DISCUSSION THREAD

**Discussion by migdur** [WinMerge/winmerge#676] [Q&A] [answered]
Hi there,

I would like to be able to compare two sets of HTML files making abstraction of differences in the HTML markup itself (so only content diffs). Is this possible at all or do I need to first html2txt all the files?

Thanks!

**Comment by sdottaka** [comment, marked as answer]
The ApacheTika plugin introduced in WinMerge 2.16.13 removes tags from HTML files and compares them.
